src/components/Value/index.tsx

```diff
@@ -0,0 +1,42 @@
+import cn from 'classnames';
+import * as React from 'react';
+
+import { Section } from 'src/components/Section';
+import { Container } from '../Container';
+
+export interface IValue {
+  title: string;
+  summary: string;
+  titleColor: string;
+}
+
+export const Value: React.FunctionComponent<IValue> = ({ title, summary, titleColor }) => {
+  return (
+    <div id={title}>
+      <h2>{title}</h2>
+      <div>{summary}</div>
+    </div>
+  );
+};
+
+export interface IValueSection {
+  values: IValue[];
+}
+export const ValueSection = ({ values }: IValueSection) => {
+  return (
+    <Section id="values" className="pt-16" noPaddingB>
```

Comment:
To be consistent with the other sections

```suggestion
    <Section id="values" {...sectionProps}>
```